src/Sylius/Behat/Page/Shop/Product/IndexPage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Shop\Product;
+
+use Sylius\Behat\Page\SymfonyPage;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class IndexPage extends SymfonyPage implements IndexPageInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isResourceOnPage($resourceName)
```

Comment:
There will be no other resources, so I'd name this ``isProductInTheList`` or something similar.